README.md

```diff
@@ -34,6 +34,24 @@ lein run bundle.tar.gz --input '{"user": {"roles": ["admin"]}}'
 Note that the above constitutes a simple flow for development and testing only. Production artifacts, or a fixed API for
 integrations, will be here at a later point in time.
 
+### Java
+
+### Evaluating an entrypint in a plan file
+
+```java
+var file = new File("path/to/my/plan.json");
```

Comment:
Neat! Do we support bundles here as well?

Reply:
Not yet. I plan to make a small PR with that support shortly. I just didn't want to further postpone getting _some_ Java docs out there.

Reply:
Sounds good!